x-pack/plugins/alerting/server/lib/alert_summary_from_event_log.ts

```diff
@@ -149,10 +150,15 @@ export function alertSummaryFromEventLog(params: AlertSummaryFromEventLogParams)
 }
 
 // return an alert status object, creating and adding to the map if needed
-function getAlertStatus(alerts: Map<string, AlertStatus>, alertId: string): AlertStatus {
+function getAlertStatus(
+  alerts: Map<string, AlertStatus>,
+  alertId: string,
+  alertUuid?: string
```

Comment:
Nit: I prefer `alertUuid: string | undefined` as it is a bit more explicit.

Reply:
There are other uses of this function which do not use the uuid, so changing to `string | undefined` would mean those uses would explicitly have to pass `undefined`.  So, seems like "optional" with `?` works better here.